src/__experimental__/components/numeral-date/numeral-date.component.js

```diff
@@ -17,6 +16,14 @@ const marginPropTypes = filterStyledSystemMarginProps(
   styledSystemPropTypes.space
 );
 
+const DATE_FORMATS = [
```

Comment:
I think that we should use regular case here, as the convention to mark constants with uppercase was used when there were no `const` keyword in js

Reply:
TBH as almost everything is `const` , this is more intuitive to me to keep this in uppercase as it shows me like "this variable keeps hardcoded values"